Automattic/wp-calypso
# Giphy: Required Branding In Sidebar

#### Changes proposed in this Pull Request

Adds the required Giphy branding in the block sidebar.



<img width="1219" alt="screen shot 2019-01-23 at 7 47 02 am" src="https://user-images.githubusercontent.com/841763/51608890-02047200-1f19-11e9-9c09-a76bead1be7c.png">


#### Testing instructions

- Spin up JN instance: https://jurassic.ninja/create?gutenpack&calypsobranch=try/giphy+branding&branch=try/giphy
- Connect Jetpack.
- Navigate to `Settings->Jetpack Constants` and enable `JETPACK_BETA_BLOCKS`
- Create Post, add Giphy block. Open the sidebar and verify the Giphy branding.
